net-misc/xmrig/xmrig-6.12.2.ebuild

```diff
@@ -18,12 +18,14 @@ fi
 
 LICENSE="Apache-2.0 GPL-3+ MIT"
 SLOT="0"
-IUSE="cpu_flags_x86_sse4_1 donate hwloc +ssl"
+IUSE="cpu_flags_x86_sse4_1 donate hwloc +ssl cuda opencl"
```

Comment:
Same as above, please keep in alphabetical order.

Reply:
I'll check, give me a little bit of time 🙂